spec/models/user_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+RSpec.describe User, type: :model do
+  let(:user_build) { build(:user) }
```

Comment:
build_stubbed better

Reply:
in user test case i have some test for invalid user, so when i change build => build_stubbed, this test not pass